cla_backend/libs/eligibility_calculator/tests/cfe_civil/test_employment.py

```diff
@@ -55,3 +55,43 @@ def test_self_employment(self):
             ],
         }
         self.assertEqual(expected, output)
+
+    def test_self_employment_drawings(self):
+        income = Income(earnings=0, self_employed=True, self_employment_drawings=20000)
+        deductions = Deductions(income_tax=0, national_insurance=0)
+        output = translate_employment(income, deductions)
+        expected = {
+            "self_employment_details": [
+                {
+                    "income": {
+                        "frequency": "monthly",
+                        "gross": 200,
+                        "tax": 0,
+                        "national_insurance": 0,
+                        "prisoner_levy": 0,
+                        "student_debt_repayment": 0
```

Comment:
as these 2 are post-MTR fields, should we be including them?

Reply:
I'm ambivanlent. is this an important blocker?

Reply:
no, just didn't know whether the press the 'approve' button or not...